crypto/transactions/builder/base.py

```diff
@@ -66,6 +67,10 @@ def multi_sign(self, passphrase, index):
 
     def schnorr_verify(self):
         return self.transaction.verify_schnorr()
+    
+    def schnorr_verify_second(self, passphrase):
+        secondPublicKey = PublicKey.from_passphrase(passphrase)
```

Comment:
This needs to take a `secondPublicKey` only as you don't know the `passphrase` of someone else. You can fetch the second public key from an address when it has one, and use that to verify the second signature. Hence why you will have to do the `PublicKey.from_passphrase(passphrase)` step in the test itself to mimic this.